gcp/container/client.go

```diff
@@ -30,16 +30,16 @@ func (c client) ListClusters() (*gcpcontainer.ListClustersResponse, error) {
 	return c.containers.List(parent).Do()
 }
 
-func (c client) DeleteCluster(zone string, cluster string) error {
-	name := fmt.Sprintf("projects/%v/locations/%v/clusters/%v", c.project, zone, cluster)
-	return c.wait(c.containers.Delete(name))
+func (c client) DeleteCluster(location string, cluster string) error {
+	name := fmt.Sprintf("projects/%v/locations/%v/clusters/%v", c.project, location, cluster)
+	return c.wait(c.containers.Delete(name), location)
 }
 
 type request interface {
 	Do(...googleapi.CallOption) (*gcpcontainer.Operation, error)
 }
 
-func (c client) wait(request request) error {
+func (c client) wait(request request, location string) error {
```

Comment:
Is the location argument being used in this wait function?

Reply:
Ive corrected for this